### PR TITLE
Declare all script targets as phony

### PR DESCRIPTION
**Pull request by skitt**
Signed-off-by: Stephen Kitt <skitt@redhat.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,5 +18,6 @@ trash-keep: .dapper
 
 .DEFAULT_GOAL := ci
 
-.PHONY: test
+.SECONDEXPANSION:
+.PHONY: $$(TARGETS)
```
Comment (tpantelis) [inline Makefile]: Forgive me - I haven't worked with Makefiles in over 10 years 😄 - just curious what is the reason for using secondary expansion here as opposed to just `.PHONY: $(TARGETS)`?

Comment (skitt) [reply Makefile]: No reason, it turns out — I was mis-remembering when secondary expansion is necessary! It’s only really useful with automatic variables (or variables which are re-set after the target definition).

**Comment by mangelajo** [comment]
@skitt there's some conflict now. Github's UI get's stuck when trying to resolve here. Can you fix it? :)

**Comment by skitt** [comment]
> @skitt there's some conflict now. Github's UI get's stuck when trying to resolve here. Can you fix it? :)

@mangelajo done!

**Pull request merged**